nomad/structs/structs.go

```diff
@@ -7829,6 +7833,8 @@ func (event *TaskEvent) PopulateEventDisplayMessage() {
 		}
 	case TaskDownloadingArtifacts:
 		desc = "Client is downloading artifacts"
+	case TaskReDownloadingArtifacts:
+		desc = "Client is re-downloading artifacts since `always_fetch` is true"
```

Comment:
I think this is a little more idiomatic for our message style.

```suggestion
		desc = "Client is re-downloading artifacts since always_fetch=true"
```